module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -482,9 +482,25 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             $itemResponse = $this->makeRequest('GET', '/item-storage/items', $query);
             $itemBody = json_decode($itemResponse->getBody());
             $notesFormatter = function ($note) {
-                return $note->note ?? '';
+                return !$note->staffOnly && $note->note ? $note->note : '';
             };
+            $textFormatter = function ($supplement) {
+                $format = '%s %s';
+                $supStat = $supplement->statement;
+                $supNote = $supplement->note;
+                $statement = sprintf($format, $supStat, $supNote);
+                return $statement ?? '';
+            };
+            $holdingNotes = array_map($notesFormatter, $holding->notes ?? []);
+            $hasHoldingNotes = !empty(implode($holdingNotes));
+            $holdingsStatements = $holding->holdingsStatements;
+            $holdingsSupplements = $holding->holdingsStatementsForSupplements;
+            $holdingsIndexes = $holding->holdingsStatementsForIndexes;
+            $issues = array_map($textFormatter, $holdingsStatements ?? []);
```

Comment:
I think it would be more clear to put the `?? []` fallback up above when initially defining `$holdingsStatements`, etc., rather than down here when you use the variables for the first time. I think there's a greater chance of PHP notices doing it this way.

Reply:
Yes, it's more concise too. This is done.